core/src/main/scala/filodb.core/memstore/TimeSeriesPartition.scala

```diff
@@ -132,6 +132,7 @@ extends ChunkMap(memFactory, initMapSize) with ReadablePartition {
       ChunkSetInfo.setChunkID(infoAddr, currentChunkID)
       ChunkSetInfo.resetNumRows(infoAddr)    // Must reset # rows otherwise it keeps increasing!
       ChunkSetInfo.setStartTime(infoAddr, ts)
+      ChunkSetInfo.setEndTime(infoAddr, ts)
```

Comment:
What's this? An unrelated mistake>

Reply:
It's an unrelated but simple fix.  Well, now that I'm looking closer at it, it's probably not necessary and can just be removed.